src/phone_number_input.rs

```diff
@@ -1,8 +1,9 @@
 use adw::prelude::ComboRowExt;
-use gtk::glib;
+use adw::traits::EntryRowExt;
```

Comment:
These can probably be merged with a single `use adw::prelude::*;`, this way `use gtk::prelude::*;` can also be removed.